test/app.ts

```diff
@@ -181,7 +197,10 @@ describe('VPN proxy', function() {
 				});
 
 			nock(`https://${BALENA_API_HOST}`)
-				.post('/v5/device(1)/canAccess', '{"action":"tunnel-8080"}')
+				.post(
+					'/v5/device(1)/canAccess',
+					'{"action":{"or":["tunnel-any","tunnel-8080"]}}',
```

Comment:
Is it not possible to pass in the JSON and have it auto-stringified?  Or if not then creating the string via an explicit `JSON.stringify` call instead may be better

Reply:
Agreed re: `JSON.stringify`, but will try passing an object too.